api/src/utils/index.ts

```diff
@@ -133,12 +140,28 @@ export const setCellHelper = (workSheet: ExcelJS.Worksheet) => ({
   row,
   column,
   value,
+  alignment,
 }: {
   row: number;
   column: string;
   value: string;
+  alignment?: ExcelJS.Alignment['horizontal'];
 }) => {
   const Row = workSheet.getRow(row);
   Row.getCell(getIndexOfAlphabet(column)).value = value;
+  if (alignment) {
+    Row.alignment = { horizontal: alignment };
+  }
   Row.commit();
 };
+
+/**
+ * concate string
+ */
+export const concat = (
```

Comment:
I don't like this function, I think it obscures what's being done. I would have only expected this to do `.join('')` given the name. I would either rename this function, or tbh I would just use `join` with `map` instead. Everyone knows what `join` and `map` do already.

Reply:
I'll rename to `mapAndJoin`, ya, because I have to reuse this function in 2 places or future, I think it's better to make it a function.